app/init.php

```diff
@@ -1214,13 +1214,17 @@ function (mixed $value) {
 
         $jwtUserId = $payload['userId'] ?? '';
         $jwtSessionId = $payload['sessionId'] ?? '';
+        $jwtSession = json_decode($payload['session'] ?? '', true);
```

Comment:
Weird... I don't think the payload has `session` It should only have `userId` and `sessionId`:

https://github.com/appwrite/appwrite/blob/e3bc1887a4abbc8de3b82d0ec7aadc5e9f46c8a1/app/controllers/api/account.php#L2380-L2389